base_layer/core/src/consensus/consensus_constants.rs

```diff
@@ -243,11 +242,9 @@ impl ConsensusConstants {
 
     /// The maximum time a block is considered to take. Used by the difficulty adjustment algorithms
     /// Multiplied by the PoW algorithm block percentage.
-    pub fn pow_max_block_interval(&self, pow_algo: PowAlgorithm) -> u64 {
-        match self.proof_of_work.get(&pow_algo) {
-            Some(v) => v.max_target_time,
-            _ => 0,
-        }
+    pub fn pow_max_block_interval(&self, pow_algo: PowAlgorithm) -> Result<u64, String> {
+        LinearWeightedMovingAverage::max_block_time(self.pow_target_block_interval(pow_algo))
```

Comment:
This seems like a big change, possibly leaking out the implementation that's abstracted by proof of work. Can you give some more reasons as to this change?

Reply:
I propose updating the comment as:
```
    /// The maximum time/interval a block is considered to take. This is used by the difficulty adjustment algorithms, 
    /// and multiplied by the PoW algorithm block percentage. For LWMA-1, `max_block_time` is a fixed multiple of 
    /// `target_block_time`, determined by `LinearWeightedMovingAverage`.
```

Reply:
After discussion with @stringhandler, accepting the suggestion to remove this function as it is easily calculated with `LinearWeightedMovingAverage::max_block_time(target_time)` and passing in `ConsensusConstants::pow_target_block_interval(..)`, as `max_block_time` is not a consensus parameter.